src/metrics/kms-service.ts

```diff
@@ -12,6 +12,16 @@ export class KMSService {
     const buffer = Buffer.from(ciphertext, "base64");
     let kms;
 
+    const region = process.env.AWS_REGION;
+    const isGovRegion = region !== undefined && region.startsWith("us-gov-");
+    let kmsClientParams = {};
+    if (isGovRegion) {
+      // Endpoints: https://docs.aws.amazon.com/general/latest/gr/kms.html
+      kmsClientParams = {
+        endpoint: `https://kms-fips.${region}.amazonaws.com`,
+      };
```

Comment:
```suggestion
      kmsClientParams['endpoint'] = `https://kms-fips.${region}.amazonaws.com`;
```

You already created the object.

Reply:
yeah but typescript. Then I'd have to do:
```typescript
    const kmsClientParams: { endpoint?: string; } = {};
    if (isGovRegion) {
      // Endpoints: https://docs.aws.amazon.com/general/latest/gr/kms.html
      kmsClientParams.endpoint = `https://kms-fips.${region}.amazonaws.com`;
    }
```

is that better performance-wise?

Reply:
Can't you just type it with the actual expected parameters type from KMS?

Reply:
No because we're importing `kmsClient` via `require` so we don't have the types.

I also see this at the top of the file
```
// In order to avoid the layer adding the 40mb aws-sdk to a deployment, (which is always available
// in the Lambda environment anyway), we use require to import the SDK.
```

so we can't import anything to have those types

Reply:
I do think this is best because it 'works' and after TS is transpiled down to JS, we no longer create a new object
```typescript
    const kmsClientParams: { endpoint?: string; } = {};
    if (isGovRegion) {
      // Endpoints: https://docs.aws.amazon.com/general/latest/gr/kms.html
      kmsClientParams.endpoint = `https://kms-fips.${region}.amazonaws.com`;
    }
```